### PR TITLE
Fix icon displaced to far right when data table cell is expanded

### PR DESCRIPTION
**Pull request by ddouglasz**
Fix icon displaced to far right when data table cell is expanded
Before: 
image

After:
image (1)

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
Thanks! 👌

**Review by emmenko — CHANGES_REQUESTED** [review]
I checked the VRTs. The column alignments (left, right) are broken now.

image

**Comment by ddouglasz** [comment]
Oh, will take a look now. Thank you.

File: packages/components/data-table/src/header-cell.styles.tsx [hunk 1]
```diff
@@ -123,8 +123,8 @@ const HeaderIconWrapper = styled.div`
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  margin-left: ${vars.spacingS};
-  padding: 0 ${vars.spacingXs};
+  margin: 0 ${vars.spacingS};
```
Comment (emmenko) [inline packages/components/data-table/src/header-cell.styles.tsx:126]: I think the margin should be left only. When the cell content is right-aligned it's visible that the space on the right is bigger than normal.

image

**Review by emmenko — COMMENTED** [review]
Thanks, just one minor thing about the `margin`.

File: packages/components/data-table/src/header-cell.styles.tsx (hunk 1 above)
Comment (ddouglasz) [reply packages/components/data-table/src/header-cell.styles.tsx]: Thank you. 
Updated here: https://github.com/commercetools/ui-kit/pull/2288/commits/5d18b00a597a722dbaf8b1b4eea7dd20f1ae9ba8

**Review by Rhotimee — APPROVED** [review]

**Review by emmenko — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**